pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/TopicStats.java

```diff
@@ -67,6 +67,7 @@ class TopicStats {
     long compactionCompactedEntriesCount;
     long compactionCompactedEntriesSize;
     StatsBuckets compactionLatencyBuckets = new StatsBuckets(CompactionRecord.WRITE_LATENCY_BUCKETS_USEC);
+    public int delayedTrackerMemoryUsage;
```

Comment:
```suggestion
    int delayedTrackerMemoryUsage;
```
do we really need public?